pixie/vm/numbers.py

```diff
@@ -135,7 +135,8 @@ def define_num_ops():
                     continue
                 extend_num_op(op, c1, c2, conv1, sym, conv2)
             extend_num_op("_quot", c1, c2, conv1, "/", conv2, wrap_start = "rt.wrap(math.floor(", wrap_end = "))")
-            extend_num_op("_rem", c1, c2, conv1, ",", conv2, wrap_start = "rt.wrap(math.fmod(", wrap_end = "))")
+            if c1 != Integer or c2 != Integer:
```

Comment:
It actually doesn't though. See, it's checking for whether either is not an integer. That is functionally identical to checking whether it is not the case that both are integers. Unless there's something else I'm not thinking of.